releases/cosmwasm/2025-03-Celo-proposal-v1.0.0.md

```diff
@@ -0,0 +1,44 @@
+# Update Celo confirmation height
+
+|                | **Owner**                              |
+| -------------- | -------------------------------------- |
+| **Created By** | @blockchainguyy <ayush@interoplabs.io> |
+| **Deployment** | @blockchainguyy <ayush@interoplabs.io> |
+
+| **Network**          | **Deployment Status** | **Date** |
+| -------------------- | --------------------- | -------- |
+| **Devnet Amplifier** | -                     | TBD      |
+| **Stagenet**         | -                     | TBD      |
+| **Testnet**          | -                     | TBD      |
+| **Mainnet**          | -                     | TBD      |
+
+## Param change instructions
+
+Create a `proposal.json` file as follows:
+
+```json
+{
+    "title": "Update Celo confirmation height",
+    "description": "Celo supports finalized tag now, therefore updating confirmation height to 2500",
```

Comment:
An approx confirmation height of 2500 is still risky, there's no guarantee it might not take longer. Should check what the param is for other chains using finalized tag for comparison